<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LDataDlg</class>
 <widget class="QDialog" name="LDataDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>268</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="shapeLabel">
         <property name="text">
          <string>&amp;Shape:</string>
         </property>
         <property name="buddy">
          <cstring>shapeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="shapeLineEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="typeLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLabel">
         <property name="text">
          <string>&amp;Label:</string>
         </property>
         <property name="buddy">
          <cstring>labelLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="linkComboBox">
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>True</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>False</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="labelLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>&amp;Type:</string>
         </property>
         <property name="buddy">
          <cstring>typeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="linkLabel">
         <property name="text">
          <string>&amp;Link:</string>
         </property>
         <property name="buddy">
          <cstring>linkComboBox</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="pathLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>NeXus &amp;Path:</string>
         </property>
         <property name="buddy">
          <cstring>pathLineEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="pathLineEdit"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>labelLineEdit</tabstop>
  <tabstop>shapeLineEdit</tabstop>
  <tabstop>typeLineEdit</tabstop>
  <tabstop>linkComboBox</tabstop>
  <tabstop>pathLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LDataDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LDataDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
